### ISSUE TITLE
Please make it possible to use with browserify-lite

### ISSUE THREAD

**Issue by josch** [webrtcHacks/adapter#612]
In Debian we don't ship browserify (yet) because of its heavy dependencies. Instead, packages are built with browserify-lite. Would it be possible to make webrtc-adapter also work with browserify-lite? Currently, when one attempts to use browserify-lite, the following error is generated:

```
$ browserify-lite ./src/js/adapter_core.js --outfile adapter.js --standalone adapter
/usr/lib/nodejs/browserify-lite/cli.js:29
  if (err) throw err;
           ^

Error: module ) in /tmp/webrtc-adapter/node-webrtc-adapter-4.1.0/src/js/chrome not found
    at trySearchPath (/usr/lib/nodejs/browserify-lite/index.js:257:32)
    at /usr/lib/nodejs/browserify-lite/index.js:262:7
    at /usr/lib/nodejs/browserify-lite/index.js:282:21
    at FSReqWrap.oncomplete (fs.js:82:15)
```

Which suggests that somehow `)` is parsed as a package name somehow.

Or is there another alternative way to bundle up the several javascript files provided by webrtc-adapter besides browserify or browserify-lite?

**Comment by fippo** [comment]
```node_modules/.bin/browserify-lite ./src/js/adapter_core.js --outfile out/lite.js```
works here with browserify-lite @ 0.5.0

**Comment by josch** [comment]
Your analysis is correct. My version of browserify-lite was just outdated. Sorry for the noise!

**Comment by josch** [comment]
Turns out that this error only occurs with browserify-lite 0.3.0 but is fixed with 0.5.0.
